include/svnlook.php

```diff
@@ -797,18 +797,25 @@ function applyGeshi($path, $filename, $language, $rev, $peg = '', $return = fals
 		}
 
 		$source = file_get_contents($filename);
+
 		if ($this->geshi === null) {
 			if (!defined('USE_AUTOLOADER')) {
 				require_once 'geshi.php';
 			}
 			$this->geshi = new GeSHi();
 		}
+
 		$this->geshi->set_source($source);
 		$this->geshi->set_language($language);
 		$this->geshi->set_header_type(GESHI_HEADER_NONE);
 		$this->geshi->set_overall_class('geshi');
 		$this->geshi->set_tab_width($this->repConfig->getExpandTabsBy());
 
+		if ($highlight == 'file') {
+			$this->geshi->enable_line_numbers(GESHI_FANCY_LINE_NUMBERS);
```

Comment:
Looking at the docs I do not fully understand fancy here: https://www.wikiparques.com/extensions/SyntaxHighlight_GeSHi/geshi/docs/geshi-doc.html#enabling-line-numbers
What is the second arguement?